### PR TITLE
Update data/portals.csv

### PR DESCRIPTION
**Pull request by will-saunders**
added State of WA geospatial and fiscal specialized open data sites

### CODE REVIEW

**Comment by rufuspollock** [comment]
@augusto-hermann can you review and then merge 😄

**Comment by augusto-herrmann** [comment]
> @augusto-hermann can you review and then merge smile

Sure. Let me just add some continuous data validation first (#186) before reviewing any further PRs to `data/portals.csv`.

File: data/portals.csv [hunk 1]
```diff
@@ -753,14 +753,16 @@ southafrica_municipal_data,"Municipal Data, South Africa",https://municipaldata.
...
 state_of_illinois,Illinois,http://data.illinois.gov/,State of Illinois,State of Illinois,,,Open Data Portal for the State of Illinois,ctic level.local unitedstates,,,"Illinois, United States","40.0796319,-89.4339809",US,en,active,2011-06-27T18:15:57.1Z,,,,
 state_of_nebraska,Nebraska,http://www.nebraska.gov/data/,State of Nebraska,State of Nebraska,,,State of Nebraska Open Data Catalog,ctic level.national unitedstates,,,"Nebraska, United States","41.7370229,-99.5873816",US,en,active,2011-06-27T18:16:01.326Z,,,,
-state_of_washington,Washington,http://data.wa.gov/,State of Washington,State of Washington,,,Public Open Data of the State of Washington,ctic unitedstates,,,"Washington, United States","47.2868352,-120.2126139",US,en,active,2011-06-27T18:16:05.479Z,,,,
```
Comment (augusto-herrmann) [inline data/portals.csv]: From the your commit's description It looks like these (763-735) are the important lines (state of WA, USA).

All other changes just alter date formatting and quotes around cells that don't really add anything meaningful. Could you please change your PR in order to include only the relevant changes to the file?

Comment (augusto-herrmann) [reply data/portals.csv]: @will-saunders, could you fix your PR to include only the addition of the state of Washington, and not changes to other lines in the file?

Comment (will-saunders) [reply data/portals.csv]: I'm not sure I can do that correctly; due to restrictions on my use of state equipment it is unlikely I will be able to install node or test adequately.  Please feel free to disregard the pull request. Is there another way to suggest changes?

Comment (will-saunders) [reply data/portals.csv]: Let me try to clarify:  I did not intend any changes to other records, but clearly they happened.  I have no way of running the code locally, so I don't know how my changes might affect other records -- apart from guessing based on the format of the file,   I know of a number of other open data portals in the state, and would be happy to contribute them if I can figure out a way to do so without affecting the other entries.

Comment (augusto-herrmann) [reply data/portals.csv]: Well, spreadsheet software do have a tendency to convert records to a csv dialect of their own preference, e.g.: adding / removing quotes, change date formats, etc. I think that may be the reason for other records being altered here.

When I just need to add some lines to a csv and don't want to mess with the rest of the file, what I usually do is:

1. make a copy of the csv,
2. open it in the spreadsheet editor (in my case, LibreOffice Calc),
3. delete all lines except the header,
4. insert the new lines,
5. save the csv,
6. open in a plain text editor (e.g. notepad, gedit),
7. copy the new lines and append them at the end of the file,
8. commit and send the PR

If you are unable to do something like this, them, when I have some time, I intend to take the lines you added and copy / paste them into a new commit to include them.

In any case, thank you for your contributions. If you have more, please send them.

**Comment by augusto-herrmann** [comment]
Hey, @will-saunders. I have just [committed](https://github.com/okfn/dataportals.org/commit/d65b9f30d41fed3d702c0d36bd15195ec15bbd07) your additions to the master branch, so we can close this PR.

Thank you for your contribution. If you have more, please send them as well.

**Pull request closed without merging**